conformance/utils/suite/suite.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package suite
+
+import (
+	"testing"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
```

Comment:
Unwanted line jump.

Reply:
This looks strange but was intentional. I'm trying to group imports into three separate categories: 

- stdlib
- other
- local

A bit more explanation of the structure here https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e. And an example of us doing similar grouping elsewhere in the project: https://github.com/kubernetes-sigs/gateway-api/blob/master/cmd/admission/main.go.

Reply:
Ah, TIL